algorithms/kernel/dtrees/gbt/classification/gbt_classification_train_dense_default_impl.i

```diff
@@ -91,8 +90,7 @@ public:
                     exp[i] = -f[i];
                     /* make all values less than threshold as threshold value
                     to fix slow work on vExp on large negative inputs */
-                    if (exp[i] < expThreshold)
-                        exp[i] = expThreshold;
+                    if (exp[i] < expThreshold) exp[i] = expThreshold;
```

Comment:
current coding guidelines say that the statement that follows conditional statement should be on a separate line.
why clang formats your code in a way that contradicts the guidelines?

@averbukh: do we need to change this in coding guidelines or clang configs?